app/models/portfolio_transaction.rb

```diff
@@ -0,0 +1,10 @@
+class PortfolioTransaction < ApplicationRecord
+  enum transaction_type: { deposit: 0, withdrawal: 1, credit: 2, debit: 3 }
```

Comment:
What would credit, debit be used for?

Reply:
credit and debit are for when students make stock purchases or chose to sell:
* debit -- stock sold
* credit -- stock bought